apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -544,3 +552,11 @@ func isContainerResourceLimitSet(ctx context.Context, namespace string, triggerT
 
 	return false
 }
+
+func getHpaName(so ScaledObject) string {
+	if so.Spec.Advanced == nil || so.Spec.Advanced.HorizontalPodAutoscalerConfig.Name == "" {
```

Comment:
```suggestion
	if so.Spec.Advanced == nil || so.Spec.Advanced.HorizontalPodAutoscalerConfig || so.Spec.Advanced.HorizontalPodAutoscalerConfig.Name == "" {
```
`HorizontalPodAutoscalerConfig` is also a pointer so I think technically this could also be `nil`

Reply:
Thanks for pointing it out, updated